src/content/en/tools/lighthouse/audits/canonical.md

```diff
@@ -0,0 +1,102 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Document doesn't have a valid rel=canonical" Lighthouse audit.
+
+{# wf_updated_on: 2018-03-06 #}
+{# wf_published_on: 2018-03-05 #}
+{# wf_blink_components: Platform>DevTools #}
+
+# Document doesn't have a valid rel=canonical  {: .page-title }
+
+## Overview {: #overview }
+
+When multiple pages have similar content, search engines consider them duplicate versions of the
+same page. For example, desktop and mobile versions of a product page are considered duplicates.
+Search engines select one of the pages as the canonical version and crawl that one more, while
+crawling the other ones less.
+
+Canonical links let you explicitly specify which version to crawl. There are multiple
+advantages to this:
+
+* You get to specify which URL should appear in search results.
+* It helps search engines consolidate multiple URLs into a single, preferred URL. For example,
+  if other sites put query parameters on the ends of links to your page, search engines
+  consolidate those URLs to your preferred version.
+* It simplifies tracking methods. Tracking one URL is easier than tracking many.
+* It improves the page ranking of syndicated content by consolidating the syndicated links to
+  your original content back to your preferred URL.
+* It optimizes crawling time. Time spent crawling duplicate pages is time not spent crawling
+  other pages with truly unique content.
+
+Sources:
+
+* [Consolidate duplicate URLs][CDU]{:.external}
+* [Learn the impact of duplicate URLs][LDU]{:.external}
+
+[CDU]: https://support.google.com/webmasters/answer/139066
+[LDU]: https://support.google.com/webmasters/answer/6080548
+
+## Recommendations {: #recommendations }
+
+Add a canonical link element to the `head` element of your HTML:
+
+    <!doctype html>
+    <html>
+      <head>
+        <link rel="canonical" href="https://example.com"/>
```

Comment:
nit: rel=canonical doesn't necessarily need to be the first tag in the document head, and it's relatively important to optimize this ordering, so could you add another `...` before this?

Reply:
✔️